application/controllers/Upload.php

```diff
@@ -11,5 +11,12 @@ function __construct()
 	public function index()
 	{
 		$this->load->view('index');
+		$this->load->view('upload');
```

Comment:
Deze regel moet je weg halen. Dit omdat de `$this->load->view('index');` er al voor zorgt dat die view word ingeladen.